CMakeLists.txt

```diff
@@ -83,10 +83,11 @@ if(NOT EXECUTORCH_ENABLE_PROGRAM_VERIFICATION)
   add_definitions(-DET_ENABLE_PROGRAM_VERIFICATION=0)
 endif()
 
-# -Os: Optimize for size. -ffunction-sections -fdata-sections: breaks function
-# and data into sections so they can be properly gc'd. -s: strip symbols
+# -O2: Moderate opt. -ffunction-sections -fdata-sections: breaks function
+# and data into sections so they can be properly gc'd. -s: strip symbol.
+# -fno-exceptions -fno-rtti: disables exceptions and runtime type.
 set(CMAKE_CXX_FLAGS_RELEASE
-    "-Os -ffunction-sections -fdata-sections -fno-exceptions -fno-rtti")
+    "-O2 -ffunction-sections -fdata-sections -fno-exceptions -fno-rtti")
```

Comment:
I'm thinking maybe we need 2 modes, one for optimize performance one for optimize size. If we optimize for perf there will be stuff like loop unrolling which actually increase binary size.